src/main/groovy/org/pih/warehouse/inventory/CycleCountStartCommand.groovy

```diff
@@ -5,16 +5,4 @@ import grails.validation.Validateable
 class CycleCountStartCommand implements Validateable {
 
     CycleCountRequest cycleCountRequest
-
-    Integer countIndex
```

Comment:
since I split the start count and start recount into separate APIs, we don't even need to specify countIndex on the start count API anymore, we can just assume countIndex==0